Core/HLE/sceMp3.cpp

```diff
@@ -397,30 +402,40 @@ static int sceMp3Init(u32 mp3) {
 	// Parse the Mp3 header
 	int layerBits = (header >> 17) & 0x3;
 	int versionBits = (header >> 19) & 0x3;
-	ctx->SamplingRate = __CalculateMp3SampleRates((header >> 10) & 0x3, versionBits);
-	ctx->Channels = __CalculateMp3Channels((header >> 6) & 0x3);
-	ctx->BitRate = __CalculateMp3Bitrates((header >> 12) & 0xF, versionBits, layerBits);
-	ctx->MaxOutputSample = CalculateMp3SamplesPerFrame(versionBits, layerBits);
-	ctx->freq = ctx->SamplingRate;
+	int bitrate = __CalculateMp3Bitrates((header >> 12) & 0xF, versionBits, layerBits);
+	int samplerate = __CalculateMp3SampleRates((header >> 10) & 0x3, versionBits);;
 
 	DEBUG_LOG(ME, "sceMp3Init(): channels=%i, samplerate=%iHz, bitrate=%ikbps, layerBits=%d ,versionBits=%d,HEADER: %08x", ctx->Channels, ctx->SamplingRate, ctx->BitRate, layerBits, versionBits, header);
```

Comment:
I don' t think this log still okay.
You have deleted ctx->SamplingRate,ctx->Channels,ctx->BitRate,ctx->MaxOutputSample,ctx->freq

Reply:
Oops, you're right.

-[Unknown]